changelog.d/7687.bugfix

```diff
@@ -0,0 +1 @@
+Correct Synapse's definition of `m.id.phone`, changing `number` to `phone`. Bug introduced in #1994.
```

Comment:
I suspect people are more interested in the version of Synapse a bug was introduced, not the PR.

```suggestion
Correct Synapse's definition of `m.id.phone`, changing `number` to `phone`. Bug introduced in v0.20.0-rc1.
```

Reply:
yes version numbers better than PR numbers, though once we get into pre-history like this "long-standing bug" is fine rather than doing archaeology.